<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdtDeviceU3606AWidget</class>
 <widget class="QWidget" name="mdtDeviceU3606AWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="cbPort"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pbScan">
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbEssais">
     <property name="text">
      <string>Essais</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lbEssais">
     <property name="text">
      <string>Default text</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbManufacturerLabel">
     <property name="text">
      <string>Manufacturer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lbManufacturer">
     <property name="text">
      <string>Unknow</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbModelLabel">
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lbModel">
     <property name="text">
      <string>Unknow</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbSerialLabel">
     <property name="text">
      <string>Serial number</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lbSerial">
     <property name="text">
      <string>Unknow</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbIfBoardFwLabel">
     <property name="text">
      <string>Interface firmware</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="lbIfBoardFw">
     <property name="text">
      <string>Unknow</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbMeasureBoardFwLabel">
     <property name="text">
      <string>Measure firmware</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="lbMeasureBoardFw">
     <property name="text">
      <string>Unknow</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lbSrcBoardFwLabel">
     <property name="text">
      <string>Source firmware</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="lbSrcBoardFw">
     <property name="text">
      <string>Unknow</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
